accumulator/forest.go

```diff
@@ -113,39 +113,40 @@ type Forest struct {
 //               forest types though (meaning there isn't functionality implemented
 //               to convert a CowForest to DiskForest and vise-versa). Pass a filepath
 //               and cowMaxCache(how much MB to use in ram) to create a CowForest.
-func NewForest(forestFile *os.File, cached bool,
-	cowPath string, cowMaxCache int) *Forest {
+
+type ForestName int
+
+const (
```

Comment:
For all the forest types here, we should copy over the comment from the `NewForest()` function and place it here.

```
  // DiskForest  - keeps the entire forest on disk as a flat file. Is the slowest
  //               of them all. Pass an os.File as forestFile to create a DiskForest.
  //
  // RamForest   - keeps the entire forest on ram as a slice. Is the fastest but
  //               takes up a lot of ram. Is compatible with DiskForest (as in you
  //               can restart as RamForest even if you created a DiskForest. Pass
  //               nil, as the forestFile to create a RamForest.
  //
  // CacheForest - keeps the entire forest on disk but caches recent nodes. It's
  //               faster than disk. Is compatible with the above two forest types.
  //               Pass cached = true to create a cacheForest.
  //
  // CowForest   - A copy-on-write (really a redirect on write) forest. It strikes
  //               a balance between ram usage and speed. Not compatible with other
  //               forest types though (meaning there isn't functionality implemented
  //               to convert a CowForest to DiskForest and vise-versa). Pass a filepath
  //               and cowMaxCache(how much MB to use in ram) to create a CowForest.
```